imports/ui/components/register/client/register.tests.js

```diff
@@ -0,0 +1,100 @@
+import {
+    name as Register
+} from '../register';
+
+import 'angular-mocks';
+
+import {
+    Meteor
+} from 'meteor/meteor';
+import {
+    Accounts
+} from 'meteor/accounts-base';
+
+import {
+    sinon
+} from 'meteor/practicalmeteor:sinon';
+
+describe('Register', function() {
+    spies.restoreAll();
+
+    if (!process.env.TESTING)
+        process.env.TESTING = 1;
+
+    beforeEach(function(done) {
+        window.module(Register);
+        done();
+    });
+
+    describe('controller', function() {
+        let controller;
+
+        beforeEach(function(done) {
+            inject(function($rootScope, $componentController) {
+                controller = $componentController(Register, {
+                    $scope: $rootScope.$new(true)
+                });
+            });
+            done();
+        });
+
+        it('should have credentials all empty by default', function(done) {
+            expect(controller.credentials).to.be.deep.equal({
+                email: '',
+                password: ''
+            });
+            done();
+        });
+
+        it('should have error empty by default', function(done) {
+            expect(controller.error).to.be.equal('');
+            done();
+        });
+
+        describe('register()', function() {
+            let validEmail = 'validEmail';
+            let validPassword = 'validPassword';
+
+            afterEach(function(done) {
+                if (spies.register)
+                    spies.register.restore();
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '{' and instead saw 'spies'.](https://www.codacy.com/app/ddspog/MAM-Seed/file/2729684824/issues/source?bid=3340041&fileBranchId=3346427#l60)